djangosaml2/views.py

```diff
@@ -378,6 +378,14 @@ def logout(request, config_loader_path=None):
 
     result = client.global_logout(subject_id)
 
+    try:
+        result = client.global_logout(subject_id)
+    except Exception as exp:
+        logger.error('Error Handled - SLO not supported by IDP: {}'.format(exp))
```

Comment:
If you decide to keep the broad exception handler, `except Exception`, it's better to use logger.exception. That way the real traceback can be known, instead of suppressed. Such broad exception handler can otherwise hide the truth of what was the real error.

Reply:
Very good suggestion, thank you. I do not love traceback into logs but It could be usefull to collect exceptions

Reply:
done, thank you